rwengine/src/engine/SaveGame.cpp

```diff
@@ -1313,13 +1313,18 @@ bool SaveGame::getSaveInfo(const std::string& file, BasicState* basicState) {
 std::vector<SaveGameInfo> SaveGame::getAllSaveGameInfo() {
     using namespace boost::filesystem;
 
-    // TODO consider windows
+#if defined(RW_WINDOWS)
+    auto homedir = getenv("USERPROFILE");
+    const char gameDir[] = "\\My Documents\\GTA3 User Files";
```

Comment:
"My Documents" is a localized name, use "Documents" instead, or what madebr said (but with FOLDERID_Documents).